src/index.js

```diff
@@ -80,7 +80,11 @@ const useReduxForm = memoize(({ layout, config }) => {
         content
       ))(
       rows.map(({ id: rowKey, cols }) => (
-        <SuiForm.Group key={rowKey} widths="equal">
+        <SuiForm.Group
+          key={rowKey}
+          widths="equal"
+          css={{ '&&.fields': { marginBottom: '0.5em' } }}
```

Comment:
Why are you hardcoding the style here? I demonstrated to you how to open style for `row` right?